1.0/docs/start/psk/creating-a-page.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: default
+type: start
+shortname: Start
+title: Creating a new page
+subtitle: Polymer Starter Kit
+---
+
+## Creating a new page
+
+This Polymer Starter Kit tutorial teaches you how to:
+
+* create a new menu item in the navigation menu
+* create content for a new page
+* route traffic to your new page
+
+This guide assumes you have already completed the [Set up guide](setting-up.html).
+
+## Setting up
+
+1. `cd` to the base directory of your project.
+
+1. Start the local development server.
+
+       grunt
+
+   The local development server opens the application in your default
+   web browser. As you modify your application, the server detects the
+   modifications, re-builds the application, and reloads your browser
+   automatically. There is no need to re-load the browser or the application.
+
+## Creating a navigation menu item
+
+1. Open `app/index.html`. 
+
+1. Find the navigation menu. The beginning of it is diplayed in the code
+   snippet below.
+
+{% highlight html %}
```

Comment:
:heavy_check_mark: fixed indentation
:x: I'm going to continue to use the highlight macro because I like to keep my files less than 80 characters wide and indenting 4 spaces gives me less space. I'm reading this as a suggestion. Let me know if you meant this as a must-change.